### PR TITLE
Trial

### PR DESCRIPTION
**Pull request by RaviH**

### CODE REVIEW

File: brew-apps.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (spencergibb) [inline brew-apps.yml]: This shouldn't be in the PR

File: config.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (spencergibb) [inline config.yml]: This shouldn't be in the PR

File: setup.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (spencergibb) [inline setup.sh]: This shouldn't be in the PR

**Comment by RaviH** [comment]
Will make  a new PR

**Pull request closed without merging**